#!/usr/bin/env python3

# {{ ansible_managed }}

from minio import Minio
from minio.error import ResponseError

downloader = Minio('{{ groups.lb.0 }}:{{ minio.tenants.two.port }}',
                 access_key='{{ minio.tenants.two.readuser }}',
                 secret_key='{{ minio.tenants.two.readpass }}',
                 secure=False)

try:
    data = downloader.get_partial_object('picture', 'bad.jpg', 1000)
    with open('good.jpg', 'wb') as file_data:
        for d in data:
            file_data.write(d)
except ResponseError as err:
    print(err)
